docs/code-snippets/free_shipping_customization.md

```diff
@@ -92,49 +90,40 @@ function hide_shipping_when_free_is_available( $rates, $package ) {
 add_filter( 'woocommerce_package_rates', 'hide_shipping_when_free_is_available', 10, 2 );
 ```
 
-### Only show free shipping in all states except…
+### Only show free shipping in all states except a few that are not eligible for free shipping.
 
 This snippet results in showing only free shipping in all states except the exclusion list. It hides free shipping if the customer is in one of the states listed:
 
 ```php
 /**
- * Hide ALL shipping options when free shipping is available and customer is NOT in certain states
+ * Show only free shipping when free shipping is available and customer is NOT in certain states, otherwise show all rates except free shipping.
+ * Change $excluded_states = array( 'CA','FL','NY' ); to include all the states that DO NOT have free shipping
```

Comment:
The snippet works following your changes, but these states were previously Alaska, Hawaii, Guam, and Puerto Rico, I think that is a more realistic use-case. Not super important so up to you if you want to change it back.

Reply:
I'll change those back, I didn't recognize the old ones, and these were easier to test, but makes more sense to remove free shipping to Alaska 👍🏼